feature/controllers/labelController.js

```diff
@@ -14,4 +28,93 @@ module.exports = {
         });
       });
   },
+
+  findOneLabel: (req, res) => {
+    const id = req.params.id;
+
+    Label.findById(id)
+      .then(label => {
+        if (!label) {
+          return res.status(404).json({
+            message: 'Label not found with id' + id,
+          });
+        }
+        res.status(200).json({ label });
+      })
+      .catch(err => {
+        if (err.kind === 'ObjectId') {
+          return res.status(404).json({
+            message: 'Label not found with id' + id,
+          });
+        }
+        return res.status(500).json({
+          message: 'Error retrieving label with id' + id,
+        });
+      });
+  },
+
+  updateLabel: (req, res) => {
+    const id = req.params.id;
+    // const issues = [];
+
+    const { name, description, color, issues } = req.body;
+    if (!name || !description || !color) {
+      return res.json({
```

Comment:
An appropriate status code should be present in responses.

Reply:
added in commit 029bc9b